src/aind_slims_api/models/instrument.py

```diff
@@ -1,13 +1,34 @@
 """Contains a model for the instrument content, and a method for fetching it"""
 
 from datetime import datetime
-from typing import Optional
+from typing import Optional, ClassVar
 
 from pydantic import Field
 
 from aind_slims_api.models.base import SlimsBaseModel
 
 
+class SlimsInstrumentRdrc(SlimsBaseModel):
+    """Model for Instrument Rdrc"""
+
+    pk: Optional[int] = Field(
+        default=None, serialization_alias="rdrc_pk", validation_alias="rdrc_pk"
+    )
+    name: Optional[str] = Field(
+        default=None, serialization_alias="rdrc_name", validation_alias="rdrc_name"
+    )
+    created_on: Optional[datetime] = Field(
+        default=None,
+        serialization_alias="rdrc_createdOn",
+        validation_alias="rdrc_createdOn",
+    )
+    _slims_table = "ReferenceDataRecord"
+    _base_fetch_filters: ClassVar[dict[str, str]] = {
+        "rdty_name": "AIND Instruments",
+    }
+
+
+# Slims Instrument model may be deprecated. New Instruments are stored in RDRC table
```

Comment:
Can we put this note in the docstring for the class? Notes in the class docstring offer easier integration with python's exosystem (sphinx, vscode extensions, etc)